packages/context/test/unit/inject.unit.ts

```diff
@@ -12,10 +12,10 @@ import {
 
 describe('function argument injection', () => {
   it('can decorate class constructor arguments', () => {
-    // tslint:disable-next-line:no-unused-variable
     class TestClass {
-      constructor(@inject('foo') foo: string) {}
+      constructor(@inject('foo') public foo: string) {}
```

Comment:
Is the following another way to silent the compiler?
```ts
constructor(@inject('foo') foo: string) {
  disableUnusedLocalError(foo);
}
```

IMO, these two lines are functionally different:
```ts
constructor(@inject('foo') foo: string) {} // foo is a local arg
constructor(@inject('foo') public foo: string) {} // foo becomes a public property of TestClass
```

Reply:
> IMO, these two lines are functionally different

Yes, those lines are different. I thought that difference does not matter in this particular unit test.

> Is the following another way to silent the compiler?

It should be, I can give it a try if we agree this is the direction to pursue.